Bio/Blast/NCBIXML.py

```diff
@@ -514,6 +514,12 @@ def _start_hit(self):
         self._hit = self._blast.alignments[-1]
 
         self._descr.num_alignments = 0
+        # Specifically ignore CREATE_VIEW statements that sometimes
+        # exist between <Hit> tags, as a result of very large remote
+        # BLAST searches.
+        if self._value.strip() == "CREATE_VIEW":
+            print(f"NCBIXML: Ignored: {self._value!r}")
```

Comment:
Rather than print, should this be ``sys.stderr.write("...\n")`` (note trailing new line needed), or use the warnings module?

Reply:
I followed what's sort of default in the rest of the module. Happy to change this to `stderr` or `warnings`, but wouldn't that make it inconsistent?

Reply:
Fine, unless we can just silently ignore the ``CREATE_VIEW`` lines?